animation_nodes/nodes/color/c_utils.pyx

```diff
@@ -0,0 +1,27 @@
+# cython: profile=True
+cimport cython
+
+from ... data_structures cimport (
+    ColorList,
+    VirtualColorList,
+    PolygonIndicesList
+)
+
+def loopsColorsList(long Amount, PolygonIndicesList polygons, VirtualColorList colors, 
+                    ColorList loopsColors):
+    cdef long i
+    for i in range(Amount):
+        loopsColors[i] = colors[polygons.indices[i]]
+    return loopsColors
+    
+def polygonsColorsList(long Amount, PolygonIndicesList polygons, VirtualColorList colors,
+                       ColorList polygonsColors):
+    cdef long i, j, index
+    cdef long polygonsAmount = len(polygons)
```

Comment:
Since it is only used once, just inline it.

Reply:
Replaced it with the inline code.